app/stacks/uk-west/appeals-service/cosmosdb.tf

```diff
@@ -11,12 +11,12 @@ resource "azurerm_cosmosdb_account" "appeals_database" {
   kind                = "MongoDB"
 
   access_key_metadata_writes_enabled = true
-  enable_automatic_failover          = true
+  automatic_failover_enabled         = true
   public_network_access_enabled      = var.database_public_access_enabled
   is_virtual_network_filter_enabled  = false
 
   # IP addresses to allow access from Azure Portal. See: https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-firewall#allow-requests-from-the-azure-portal
-  ip_range_filter = var.database_public_access_enabled ? null : "104.42.195.92,40.76.54.131,52.176.6.30,52.169.50.45,52.187.184.26"
+  ip_range_filter = var.database_public_access_enabled ? null : ["104.42.195.92, 40.76.54.131, 52.176.6.30, 52.169.50.45, 52.187.184.26"]

```

Comment:
this should be an array of strings, rather than all in one string:

`["104.42.195.92", "40.76.54.131", "52.176.6.30", "52.169.50.45", "52.187.184.26"]`